src/strategies/index.ts

```diff
@@ -230,6 +230,7 @@ import * as bottoDao from './botto-dao';
 import * as genart from './genart';
 import * as zorro from './zorro';
 import * as voltVotingPower from './volt-voting-power';
+import * as balancerPoolid from './balancer-poolid';
```

Comment:
Hi @jo-chemla Rename the folder name to lower case, like here `balancer-poolid`

Reply:
Ah I did that but it seems like github did not handle it

Reply:
Ok that's done thanks!